core/controllers/classroom.py

```diff
@@ -35,6 +35,9 @@ def get(self):
         if not constants.ENABLE_NEW_STRUCTURE_PLAYERS:
             raise self.PageNotFoundException
 
+        if not constants.SHOW_CLASSROOM:
```

Comment:
You can maybe combine this with the above condition in one 'if' statement?

Reply:
Done.